cmd/cloud-connector/main.go

```diff
@@ -23,6 +23,14 @@ func NewRootCommand() *cobra.Command {
 		Use: "cloud-connector",
 	}
 
+	var connectionCountCmd = &cobra.Command{
+		Use:   "connection_count",
```

Comment:
I know its overly verbose (and nit-picky) but can you change the command name to "publish_connection_count"?